tailwind.config.js

```diff
@@ -61,6 +61,7 @@ export default {
                     muted: color('--background-muted', colors.slate[50]),
                 },
                 backdrop: color('--backdrop', 'rgba(0, 0, 0, 0.4)'),
+                success: color('--success', '#00B67A'),
```

Comment:
Maybe just use a green color from Tailwind somewhere? If we're going this way you may need a background, border, etc later, see: https://github.com/rapidez/core/blob/0470c917a82036e871f1547916acb787493d512a/config/rapidez/frontend.php#L82